test/fixtures/shopifolk_correct.conf

```diff
@@ -0,0 +1,3 @@
+[core]
+account = test@shopify.com
+project = shopify-dev
```

Comment:
Minor: can you set your editor to prevent those end-of-line warnings? https://thoughtbot.com/blog/no-newline-at-end-of-file